include/deal.II/fe/fe_tools.h

```diff
@@ -977,6 +977,37 @@ namespace FETools
                           const bool do_tensor_product = true);
 
     /**
+     * Same as above for an arbitrary number of parameters of type
+     * <code>std::pair<std::unique_ptr<FiniteElement<dim, spacedim>>, unsigned int></code>
+     * and <code>do_tensor_product = true</code>.
+     */
+    template <int dim, int spacedim, class... FESystems,
+              typename = typename std::enable_if<all_same_as<typename std::decay<FESystems>::type...,
+                                                             std::pair<std::unique_ptr<FiniteElement<dim, spacedim>>,
+                                                                 unsigned int>>::value>::type>
```

Comment:
at this point you've already verified that the arguments are all `std::pair<...,...>` because you call this function from `FESystem`. So I think here it would make sense to take a `std::initializer_list` (though, would that involve copying the `std::unique_ptr`s?).

Reply:
I don't have a version to make this work with `std::initializer_list` yet. I can have a look next week.